python/lsst/ap/pipe/make_apdb.py

```diff
@@ -73,7 +73,8 @@ def parse_args(self, args=None, namespace=None):
         """
         if not namespace:
             namespace = argparse.Namespace()
-        namespace.config = ApdbConfig()
+        # Make ApdbConfig as a subconfig of DiaPipelineConfig to ensure correct defaults get set
+        namespace.config = DiaPipelineConfig().apdb.value
```

Comment:
Can you please clarify for me what this is doing? I got the impression that `ApdbConfig`, `ApPipeConfig`, and `DiaPipelineConfig` are all entirely separate now, but this seems to imply one depends on another.

Reply:
`ApdbConfig` can stand alone, but its default config isn't what AP actually uses. I needed to make sure [these defaults](https://github.com/lsst/ap_association/blob/master/python/lsst/ap/association/diaPipe.py#L145-L150) got called, because otherwise you get nasty errors about missing columns. This code is a bit of a hack, I just couldn't think of another way of doing things short of telling the user to configure `DiaPipelineConfig` (which would be the worst of both worlds, IMO).